development/compiling/compiling_for_osx.rst

```diff
@@ -53,9 +63,14 @@ editor binary built with ``scons p=osx target=release_debug``::
 
     cp -r misc/dist/osx_tools.app ./Godot.app
     mkdir -p Godot.app/Contents/MacOS
-    cp bin/godot.osx.tools.64 Godot.app/Contents/MacOS/Godot
+    cp bin/godot.osx.tools.universal Godot.app/Contents/MacOS/Godot
```

Comment:
In the paragraph above, `scons p=osx target=release_debug` is no longer accurate if we're using the universal version as the reference. I'd suggest to replace it by "built with ``target=release_debug``" only.

Reply:
and its `godot.osx.opt.tools.universal` in case of `target=release_debug`